src/amp_kart_bringup/params/nav2.params.yaml

```
@@ -87,77 +87,49 @@ bt_navigator_rclcpp_node:
 
 controller_server:
   ros__parameters:
-    use_sim_time: True
     controller_frequency: 20.0
     min_x_velocity_threshold: 0.001
     min_y_velocity_threshold: 0.5
     min_theta_velocity_threshold: 0.001
     progress_checker_plugin: "progress_checker"
-    goal_checker_plugin: "goal_checker"
+    goal_checker_plugins: "goal_checker"
```

```suggestion
    goal_checker_plugin: "goal_checker"
```
shouldn't be changed